### PR TITLE
feat(GKEE): update cluster observability queries and docs

### PR DESCRIPTION
**Pull request by apeabody**

### CODE REVIEW

File: dashboards/google-kubernetes-engine-enterprise/README.md [hunk 1]
```diff
@@ -10,14 +10,14 @@
 |Google Kubernetes Engine (GKE) Enterprise Cluster Observability CPU|
 |:---------------------|
 |Filename: [gke-enterprise-cluster-observability-cpu.json](gke-enterprise-cluster-observability-cpu.json)|
-|This dashboard has 10 charts using the related [Google Kubernetes Engine (GKE) metrics](https://cloud.google.com/monitoring/api/metrics_kubernetes) and [GKE attached clusters metrics](https://cloud.google.com/kubernetes-engine/multi-cloud/docs/attached/generic/reference/versions/v1.29.0-gke.2/metrics).|
+|This top 5 cluster dashboard has 10 charts using the related [Google Kubernetes Engine (GKE) metrics](https://cloud.google.com/monitoring/api/metrics_kubernetes) and [GKE attached clusters metrics](https://cloud.google.com/kubernetes-engine/multi-cloud/docs/attached/generic/reference/versions/v1.29.0-gke.2/metrics).|
```
Comment (johnbryan) [inline dashboards/google-kubernetes-engine-enterprise/README.md:13]: maybe include "CPU" and "Memory" somewhere in these descriptions?

eg. "...has 10 charts about CPU utilization, using the related..."

File: dashboards/google-kubernetes-engine-enterprise/gke-enterprise-cluster-observability-cpu.json [hunk 2]
```diff
@@ -103,7 +103,7 @@
...
+                  "timeSeriesQueryLanguage": "fetch k8s_container\n| { { metric kubernetes.io/container/cpu/request_cores\n    ; metric kubernetes.io/anthos/container/cpu/request_cores }\n    | union\n  ; { metric kubernetes.io/container/cpu/core_usage_time\n    ; metric kubernetes.io/anthos/container/cpu/core_usage_time }\n    | union\n    | rate\n    | every 1m }\n| align next_older(2m)\n| group_by [resource.location, resource.cluster_name], .sum()\n| map\n    update[\n      resource.cluster_name: re_extract(resource.cluster_name, '(?:.+/|^)(.+)')]\n| join\n| sub\n| top 55",
```
Comment (johnbryan) [inline dashboards/google-kubernetes-engine-enterprise/gke-enterprise-cluster-observability-cpu.json:106]: looks like this became `top 55` by mistake

File: dashboards/google-kubernetes-engine-enterprise/gke-enterprise-cluster-observability-overview.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
...
+                  "timeSeriesQueryLanguage": "fetch k8s_container\n| { { metric kubernetes.io/container/cpu/core_usage_time\n    ; metric kubernetes.io/anthos/container/cpu/core_usage_time }\n    | union\n    | rate\n    | every 1m\n    | align next_older(2m)\n  ; { metric kubernetes.io/container/cpu/request_cores\n    ; metric kubernetes.io/anthos/container/cpu/request_cores }\n    | union\n    | align next_older(2m) }\n| group_by [resource.location, resource.cluster_name], .sum()\n| map\n    update[\n      resource.cluster_name:\n        re_extract(resource.cluster_name, '(?:.+/|^)(.+)', '\\\\1')]\n| outer_join 0\n| div\n| top 5\n| scale \"%\"",
```
Comment (johnbryan) [inline dashboards/google-kubernetes-engine-enterprise/gke-enterprise-cluster-observability-overview.json:21]: FMI, why did you add `\\\\1` in the regex extracts? Is that a functional change or just being explicit about which capturing group it's using?

File: dashboards/google-kubernetes-engine-enterprise/README.md (hunk 1 above)
Comment (johnbryan) [reply dashboards/google-kubernetes-engine-enterprise/README.md:13]: also in metadata

**Review by johnbryan — CHANGES_REQUESTED** [review]

File: dashboards/google-kubernetes-engine-enterprise/gke-enterprise-cluster-observability-overview.json (hunk 3 above)
Comment (apeabody) [reply dashboards/google-kubernetes-engine-enterprise/gke-enterprise-cluster-observability-overview.json:21]: Hmm @johnbryan  - I didn't actually intend that change, perhaps related to auto-correct/formatter? I'll double check against my model implementation.

File: dashboards/google-kubernetes-engine-enterprise/README.md (hunk 1 above)
Comment (apeabody) [reply dashboards/google-kubernetes-engine-enterprise/README.md:13]: Sounds good - since these should be mostly done, I'll go back and update them en mass.

File: dashboards/google-kubernetes-engine-enterprise/gke-enterprise-cluster-observability-cpu.json (hunk 2 above)
Comment (apeabody) [reply dashboards/google-kubernetes-engine-enterprise/gke-enterprise-cluster-observability-cpu.json:106]: Yeah, for some reason the mql editor did that sometimes on paste/enter.  I thought I caught all of them, but thanks for catching this one that slipped through!

File: dashboards/google-kubernetes-engine-enterprise/README.md (hunk 1 above)
Comment (apeabody) [reply dashboards/google-kubernetes-engine-enterprise/README.md]: complete

File: dashboards/google-kubernetes-engine-enterprise/gke-enterprise-cluster-observability-overview.json (hunk 3 above)
Comment (apeabody) [reply dashboards/google-kubernetes-engine-enterprise/gke-enterprise-cluster-observability-overview.json]: reverted these out

**Review by johnbryan — APPROVED** [review]
